packages/components/src/navigable-container/test/navigable-menu.tsx

```diff
@@ -34,6 +35,7 @@ describe( 'NavigableMenu', () => {
 		// Mocking `getClientRects()` is necessary to pass a check performed by
 		// the `focus.tabbable.find()` and by the `focus.focusable.find()` functions
 		// from the `@wordpress/dom` package.
+		// @ts-expect-error TODO: Don't know how to resolve
```

Comment:
This is hacky. It Prevents:

```
packages/components/src/navigable-container/test/navigable-menu.tsx:38:3 - error TS2322: Type '() => string[]' is not assignable to type '() => DOMRectList'.
  Property 'item' is missing in type 'string[]' but required in type 'DOMRectList'.

38   window.HTMLElement.prototype.getClientRects = function () {
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

  node_modules/typescript/lib/lib.dom.d.ts:4182:5
    4182     item(index: number): DOMRect | null;
             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    'item' is declared here.

```

Reply:
Same as commented for `TabbableContainer`'s tests

Reply:
Thanks, committed in https://github.com/WordPress/gutenberg/pull/49377/commits/049f3dccb44d7ac24b2b67d3b4f48241727e6a25